src/payment/coupon/CouponForm.jsx

```diff
@@ -96,19 +96,19 @@ export class CouponForm extends Component {
 
     if (benefitType === 'Absolute') {
       return (
-        <FormattedMessage
-          id="payment.coupon.benefit.absolute"
-          defaultMessage="Coupon {code} applied for {amount} off"
-          description="A coupon has been applied for a fixed currency discount, like $10.  Currency symbol will already be provided."
-          values={{
-            code,
-            amount: (
-              <LocalizedPrice amount={benefitValue} />
-            ),
-          }}
-        >
-          {renderMuted}
```

Comment:
I would either:
1. Delete `renderMuted` and switch the other usage to match this pattern, or
2. Either inline or move `renderMuted` to its lone usage, rather than the top of the file, since it is no longer shared.